spec/features/index_page/user_analytics_spec.rb

```diff
@@ -54,4 +54,8 @@
   scenario 'tracks time period reveal' do
     expect(page).to have_selector('[data-gtm-id="time-period-options"] summary')
   end
+
+  scenario 'tracks clicks on app name in header' do
```

Comment:
An explicit mention of the tracking being in Google Analytics, or Google Tag Manager (wherever it is) might be good, otherwise someone could see this as a redundant test.

Reply:
All the tests in this spec are for making sure our dependencies in our codebase for analytics and user tracking are present. It'd be weird to be more specific for this particular scenario. Not sure I understand why someone would see it as redundant?

Reply:
Ah, yeah, I just looked at the diff, I didn't actually notice this was a feature test or what file it was in. In that case it's fine :+1: